instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from 'react';
+import "./CommentSection.css"
+import Comments from './Comments'
+
+
+
+class CommentSection extends Component {
```

Comment:
This functionality and data could be lifted to your App.js and you'd have a single source of truth for all your information.  Again, your solution is valid but I would encourage you to try and lift as much as possible.